README.md

```diff
@@ -1,14 +1,120 @@
 
 # customs-financials-session-cache
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90-green.svg)](/target/scala-3.3.5/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
 This repository is responsible for storing the duty deferment account number and associated data via a UUID (LinkId) to prevent the passing over a URL parameter
 
-| Path                               | Description                                          |
-| ---------------------------------  | ---------------------------------------------------- |
-| GET /account-link/:id/:linkId | Retrieves the AccountLink based on the sessionId (id) and the LinkId (linkid)|
-| POST /update-links | Stores the generated AccountLinks into mongo |
-| DELETE /remove/:id  | Removes all associated links for a given EORI based on session id |
+## Running the service
+
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9840
+
+### Required dependencies
+
+There are a number of dependencies required to run the service.
+
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                       | Description                                                      |
+|-----------------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`          | Runs all dependencies                                            |
+| `sm2 -s`                                      | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_SESSION_CACHE` | Stop the micro service                                           |
+| `sbt run` or ` sbt "run 9840"`                | (from root dir) to compile the current service with your changes |
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
+
+* `AUTH`
+* `AUTH_LOGIN_STUB`
+* `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
+* `USER_DETAILS`
+
+### Login enrolments
```

Comment:
Better to remove this section entirely as its a backend service and no endpoint requires the Enrolments